.eslintrc.json

```diff
@@ -108,7 +108,7 @@
         "max-classes-per-file": "off",
         "max-depth": "error",
         "max-len": "off",
-        "max-lines": "error",
+        "max-lines": "off",
```

Comment:
What is the rationale for this?

Reply:
Most of the files in this package are over 300 lines (arbitrary) so this causes errors that aren't actually errors. I'm open to suggestion -- this was easier than saying ignore this particular rule in each file.